docs/user/FlowVariables.md

```diff
@@ -0,0 +1,268 @@
+
+
+| Variable       | Description                                                                                                   |
+|----------------|---------------------------------------------------------------------------------------------------------------|
+| `CTS_BUF_CELL` | The buffer cell used in the clock tree.                                                                       |
+| `FILL_CELLS`   | Filler cells list used to fill any spaces between regular library cells to avoid planarity problems post-CTS. |
+
+
+### Routing
+
+
+| Variable            | Description                                         |
+|---------------------|-----------------------------------------------------|
+| `MIN_ROUTING_LAYER` | The number of lowest layers to be used in routing.  |
+| `MAX_ROUTING_LAYER` | The number of highest layers to be used in routing. |
+
+
+### Extraction
+
+
+| Variable            | Description                                           |
+|---------------------|-------------------------------------------------------|
+| `RCX_RULES`         | RC Extraction rules file path.                        |
+| `SET_RC_TCL`        | Metal & Via RC definition file path.                  |
+| `FILL_CONFIG`       | JSON rule file for metal fill during chip finishing.  |
+| `KLAYOUT_TECH_FILE` | A mapping from LEF/DEF to GDS using the KLayout tool. |
+
+
+## Design Specific Configuration Variables
+
+
+### Required Variables
+
+
+Following minimum design variables must be defined in the design configuration
+file for each design located in the OpenROAD-flow-scripts directory of
+`./flow/designs/<PLATFORM>/<DESIGN_NAME>/config.mk`
+
+
+| Variable        | Description                                                                                                                                                          |
+|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `PLATFORM`      | Specifies process design kit or technology node to be used.                                                                                                          |
+| `DESIGN_NAME`   | The name of the top-level module of the design.                                                                                                                      |
+| `VERILOG_FILES` | The path to the design Verilog files.                                                                                                                                |
+| `SDC_FILE`      | The path to design constraint (SDC) file.                                                                                                                            |
+| `CLOCK_PORT`    | The name of the design's clock port used in Static Timing Analysis. If your design does not have a clock port defined as an empty variable `export CLOCK_PORT= " "`. |

```

Comment:
should there really be a white space inside the quotes?

Reply:
its an empty string